server/src/__tests__/user.test.integration.ts

```diff
@@ -20,8 +24,8 @@ describe('User Router', () => {
     DB.usersEntityRepo = DB.orm.em.getRepository(UserEntity)
   })
 
-  afterAll(() => {
-    DB.orm.close()
+  afterAll(async () => {
+    await DB.orm.close()
   })
```

Comment:
```suggestion
  afterAll(DB.orm.close)
```

Reply:
DB.orm.close doesn't work as a callback. The error if you're curious:

Argument of type '(force?: boolean | undefined) => Promise<void>' is not assignable to parameter of type 'ProvidesCallback'.
  Types of parameters 'force' and 'cb' are incompatible.
    Type 'DoneCallback' is not assignable to type 'boolean | undefined'.
      Type 'DoneCallback' is not assignable to type 'true'.